studio/components/layouts/AppLayout/EnableBranchingButton/BranchingWaitlistPopover.tsx

```diff
@@ -35,16 +39,16 @@ const BranchingWaitlistPopover = () => {
           enrolled in database branching.
         </p>
         <div className="flex items-center space-x-2">
-          <Link passHref href={'/'}>
+          <Link passHref href="https://forms.supabase.com/branching-request">
             <a rel="noreferrer" target="_blank">
               <Button>Join waitlist</Button>
             </a>
           </Link>
-          <Link passHref href={'/'}>
+          {/* <Link passHref href={'/'}>
             <a rel="noreferrer" target="_blank">
               <Button type="default">View the docs</Button>
             </a>
-          </Link>
+          </Link> */}
```

Comment:
Still needed?

Reply:
i'm leaving these as placeholders cause we should have docs for branching eventually